lib/BackgroundJob/RetryJob.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
+ * @copyright Copyright (c) 2021 Gary Kim <gary@garykim.dev>
+ *
+ * @author Bjoern Schiessle <bjoern@schiessle.org>
+ * @author Björn Schießle <bjoern@schiessle.org>
+ * @author Joas Schilling <coding@schilljs.com>
+ * @author Lukas Reschke <lukas@statuscode.ch>
+ * @author Morris Jobke <hey@morrisjobke.de>
+ * @author Roeland Jago Douma <roeland@famdouma.nl>
+ * @author Gary Kim <gary@garykim.dev>
```

Comment:
is it so much copy paste?

Reply:
It's mostly the same as https://github.com/nextcloud/server/blob/master/apps/federatedfilesharing/lib/BackgroundJob/RetryJob.php so just kept all the author names.